frontend/src/pages/Home.jsx

```diff
@@ -2,14 +2,30 @@ import React, { Component } from "react";
 import HomeContent from "../components/HomeContent/homeContent";
 import { connect } from "react-redux";
 import { bindActionCreators } from "redux";
-// import { Link } from "react-router-dom";
+import {
+  setLogin_email,
+  setLogin_password,
+  setRegister_email,
+  setRegister_password,
+  set_secondRegister_password,
+  setLogin_dialog,
+  setRegister_dialog,
+} from "../actions/Home";
 import logo from "../resources/logo.png";
```

Comment:
picky: If this is the original ReactJS image, since we're not using it anymore, we can probably remove this.